activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+*   `define_attribute_methods` defines alias attribute methods along with actual attribute methods
+
+    `define_attribute_methods` can and should be used to bring back previously undefined alias attribute methods
```

Comment:
>because both were in main and never in a release.

You are right and for an application that runs on a stable Rails version the bug was never there. I think we may want to make sure it is clear that `define_attribute_methods` is supposed to be used to define alias attribute methods if previously removed but that could be done through documentation

I'll close this PR